ClinGen/clincoded
# Connects to 1837. Added affiliation and R24 new users.

Steps to test #1837:
1. Update a user, adding affiliation 10033.
2. Logout and log back in as the user to confirm that "Germline/Somatic Variant Subcommittee" appears in the affiliation modal.
